game-core/src/main/java/games/strategy/triplea/delegate/battle/casualty/CasualtySelector.java

```diff
@@ -346,6 +341,29 @@ private static void killAmphibiousFirst(final List<Unit> killed, final Collectio
     return Tuple.of(defaultCasualtySelection, sorted);
   }
 
+  public static List<Unit> getSortedUnits(
+      final Collection<Unit> targetsToPickFrom,
```

Comment:
Does it make sense to add a helper class to hold these parameters? Is the same list of params passed around to many other functions too?

Reply:
I can't find any other place besides here and the battletree calculator that would use this same list.

Reply:
I'm not even sure if I should have created this function since it just creates a parameter object.  And I believe the parameter class was created so the function wouldn't have so many arguments.  So making a new function to create the parameter object kind of defeats the purpose of the parameter object.